### PR TITLE
Remove MV2 code from "Create an extension tutorial, part 2"

### PR DESCRIPTION
**Pull request by bhuvanapriyap**
Rendered article for review:
* **Create an extension tutorial, part 2**
   * https://review.learn.microsoft.com/microsoft-edge/extensions-chromium/getting-started/part2-content-scripts?branch=pr-en-us-3278
   * [Before/Live](https://learn.microsoft.com/microsoft-edge/extensions-chromium/getting-started/part2-content-scripts)
   * Removed MV2 code.
   * Removed MV3/MV2 tabset tagging.

AB#54090967

### CODE REVIEW

**Review by mikehoffms — APPROVED** [review]
LGTM - Writer/Editor

**Review by captainbrosset — APPROVED** [review]

File: microsoft-edge/extensions-chromium/getting-started/part2-content-scripts.md [hunk 1]
```diff
@@ -6,13 +6,11 @@ ms.author: msedgedevrel
...
 ms.subservice: extensions
-ms.date: 10/26/2022
+ms.date: 09/24/2024
 ---
 # Create an extension tutorial, part 2
 
-To see the completed extension package source for this part of the tutorial, go to [MicrosoftEdge-Extensions repo > extension-getting-started-part2](https://github.com/microsoft/MicrosoftEdge-Extensions/tree/main/Extension%20samples/extension-getting-started-part2/extension-getting-started-part2).  
-
-The source code has been updated from Manifest V2 to Manifest V3.
+To see the completed extension package source for this part of the tutorial, go to [MicrosoftEdge-Extensions repo > extension-getting-started-part2](https://github.com/microsoft/MicrosoftEdge-Extensions/tree/main/Extension%20samples/extension-getting-started-part2/extension-getting-started-part2).  The source code uses Manifest V3.
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/getting-started/part2-content-scripts.md:13]: This Part 2 PR (& the Part 1 PR) is blocked by needing to update demo from MV2 to MV3.
@bhuvanapriyap @captainbrosset

Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/getting-started/part2-content-scripts.md:13]:
```suggestion
To see the completed extension package source for this part of the tutorial, go to [MicrosoftEdge-Extensions repo > extension-getting-started-part2](https://github.com/microsoft/MicrosoftEdge-Extensions/tree/main/Extension%20samples/extension-getting-started-part2/extension-getting-started-part2).
```
i doubt this article's sentences should mention MV2 or MV3 at all. The demo code already states "mv3".

Comment (mikehoffms) [reply microsoft-edge/extensions-chromium/getting-started/part2-content-scripts.md:13]: Extensions repo Part 2 already says MV3:
https://github.com/microsoft/MicrosoftEdge-Extensions/blob/main/Extension%20samples/extension-getting-started-part2/extension-getting-started-part2/manifest.json
`"manifest_version": 3,`

Comment (mikehoffms) [reply microsoft-edge/extensions-chromium/getting-started/part2-content-scripts.md]: removed statement about whether Extensions repo Part 2 uses MV2 or MV3.

Extensions repo Part 2 already says MV3, and always said that:
https://github.com/microsoft/MicrosoftEdge-Extensions/blob/main/Extension%20samples/extension-getting-started-part2/extension-getting-started-part2/manifest.json
`"manifest_version": 3,`

**Review by bhuvanapriyap — COMMENTED** [review]
LGTM - PM

**Pull request merged**